hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestEndpoints.java

```diff
@@ -366,25 +370,19 @@ public void setUp() throws Exception {
     } catch (Exception ex) {
       Assertions.fail(ex.getMessage());
     }
-
     // Write Data to OM
-
     // A sample volume (sampleVol) and a bucket (bucketOne) is already created
     // in AbstractOMMetadataManagerTest.
     // Create a new volume and bucket and then write keys to the bucket.
     String volumeKey = reconOMMetadataManager.getVolumeKey("sampleVol2");
     OmVolumeArgs args =
         OmVolumeArgs.newBuilder()
-            .setVolume("sampleVol2")
-            .setAdminName("TestUser")
-            .setOwnerName("TestUser")
-            .build();
+            .setVolume("sampleVol2").setAdminName("TestUser")
+            .setOwnerName("TestUser").build();
     reconOMMetadataManager.getVolumeTable().put(volumeKey, args);
 
     OmBucketInfo bucketInfo = OmBucketInfo.newBuilder()
-        .setVolumeName("sampleVol2")
-        .setBucketName("bucketOne")
-        .build();
+        .setVolumeName("sampleVol2").setBucketName("bucketOne").build();
```

Comment:
here too.

Reply:
Got It! 
Thanks weichu I have made the changes